test/src/test/kotlin/graphql/nadel/tests/EngineTests.kt

```diff
@@ -35,7 +35,7 @@ import java.util.concurrent.CompletableFuture
  * 2. Test name e.g. hydration inside a renamed field
  * 3. Copy paste output from selecting a test in the IntelliJ e.g. java:test://graphql.nadel.tests.EngineTests.current hydration inside a renamed field
  */
-private val singleTestToRun = (System.getenv("TEST_NAME") ?: "")
+private val singleTestToRun = (System.getenv("TEST_NAME") ?: "basic hydration")
```

Comment:
I think this is bad - it will run only that test instead of all of them ??

Reply:
whoops, was not supposed to push that